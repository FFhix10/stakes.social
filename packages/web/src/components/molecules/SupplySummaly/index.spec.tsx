import React from 'react'
import { render } from '@testing-library/react'
import { SupplySummary } from '.'
import 'src/__mocks__/window/matchMedia.mock'
import BigNumber from 'bignumber.js'

describe(`${SupplySummary.name}`, () => {
  test('Snapshot', () => {
    const component = render(<SupplySummary apy={new BigNumber(1)} annualSupplyGrowthRatio={new BigNumber(1)} />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
