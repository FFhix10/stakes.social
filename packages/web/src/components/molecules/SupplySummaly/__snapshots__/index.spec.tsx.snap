// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupplySummary Snapshot 1`] = `
<body>
  <div>
    <div
      class="re975l-7 JPStL"
    >
      <h2>
        Current Staking Rewards
      </h2>
      <div
        class="re975l-5 dqaFMD"
      >
        <div
          class="re975l-6 jnaSZR"
        >
          <span
            class="re975l-2 bQJvIN"
          >
            1
            %
          </span>
          <label>
            Stakers APY
          </label>
        </div>
        <div
          class="re975l-6 jnaSZR"
        >
          <span
            class="re975l-2 bQJvIN"
          >
            %
          </span>
          <label>
            Creators APY
          </label>
          <a
            href="https://docs.devprtcl.com/protocol/tokenomics"
            rel="noopener noreferrer"
            target="_blank"
          >
            Learn more
          </a>
        </div>
        <div
          class="re975l-6 jnaSZR"
        >
          <span
            class="re975l-2 bQJvIN"
          >
            1
            %
          </span>
          <label>
            DEV ASG
          </label>
        </div>
      </div>
      <hr
        color="lightgrey"
      />
    </div>
  </div>
</body>
`;
